rdrf/rdrf/helpers/registry_features.py

```diff
@@ -19,3 +19,4 @@ class RegistryFeatures:
     CONDITIONAL_RENDERING_DISABLED = "conditional_rendering_disabled"
     XNAT_INTEGRATION = "xnat_integration"
     LONGITUDINAL_FOLLOWUPS = "longitudinal_followups"
+    PATIENT_EMAIL_ACTIVATION_OPTIONAL_FOR_ADMIN = "patient_email_activation_optional_for_admin"
```

Comment:
Need to check with Ryan if this feature is necessary.

Reply:
As per discussion:
1. We still want to keep this flag, but turn it on by default for all registries
2. Swap the radio buttons on the 'Email Change request page' so that the 'Complete email change without requiring activation from the user' is listed first.